api.raml

```diff
@@ -0,0 +1,418 @@
+#%RAML 1.0
+title: REST API doc for token.store
+mediaType: application/json
+baseUri: https://v1.api.token.store
+
+types:
+  SortOrder:
+    type: string
+    description: Order
+    enum: [desc, asc]
+  Address:
+    type: string
+    pattern: /^(0x)?[0-9a-f]{40}$/
+/ticker:
+  get:
+    description: Returns 24h trade volume and the last price for each coin.
+    responses:
+      200:
+        body:
+          example: |
+            {
+              "ETH_TTT":{
+                "tokenAddr":"0xf3bc4a5abbb9ffa56c84d27f167e7257c64bd0b2",
+                "quoteVolume":18,
+                "baseVolume":6.050000000000001,
+                "last":0.15,
+                "bid":0.3,
+                "ask":0.15,
+                "percentChange":-90.54688681639148
+              }
+            }
+/trades:
+  get:
+    description: Historical trades Return historical trades executed. Trades returned in ascending order. Option available to query historical trades based on tradeID, default is to return latest 100 trades
+    queryParameters:
+      pair:
+        type: string
+        description:
+        example: ETH_EOS
+      fromId:
+        type: number
+        description: query by tradeID, display trades after this ID inclusive (optional, default to return latest trades)
+        required: false
+      limit:
+        type: number
+        description: number of trades returned (optional, default to 100)
+        default: 100
+        example: 100
+        required: false
+      order:
+        type: SortOrder
+        required: false
+    responses:
+      200:
+        body:
+          example: |
+            [
+             {
+                "id":123,
+                "timestamp":1520797156,
+                "createdAt":"2018-03-11T19:39:16.967Z",
+                "baseAmount":0.01,
+                "price":2,
+                "type":"sell",
+                "tokenGet":"ETH",
+                "tokenGive":"EOS"
+             },
+             {
+                "id":123,
+                "timestamp":1520797166,
+                "createdAt":"2018-03-11T19:39:26.011Z",
+                "baseAmount":0.01,
+                "price":2,
+                "type":"sell",
+                "tokenGet":"ETH",
+                "tokenGive":"EOS"
+             }
+            ]
+/pairs:
+  get:
+    description: Get list of pairs
+    responses:
+      200:
+        body:
+          example: |
+            [
+             {
+                "id":"ETH_EGR",
+                "fromSymbol":"ETH",
+                "toSymbol":"EGR"
+             },
+             {
+                "id":"ETH_BAS",
+                "fromSymbol":"ETH",
+                "toSymbol":"BAS"
+             }
+            ]
+/orderbook:
```

Comment:
We need to discuss how we position it in v1.1 - previously we wanted to fix it (converting arrays to dictionaries so it's extensible etc). I think it may be worth keeping it as a shorter/readable version of new orders.

Let's probably do that and let's probably not increment API version for now, this is still 1.0, we're not changing old endpoints, we're creating new ones. And then we do 1.1 and update any existing endpoints if needed.

Reply:
So yeah let's not break existing endpoints, extend 1.0 with the new ones and then have a separate iteration for 1.1